### PR TITLE
Sam/styling - *don't* accept this PR!

### PR DESCRIPTION
**Pull request by srsexton94**
Hey yall so I'm running into some significant bugs with this and could use your help if you have the time

Primarily my biggest bug is that update photo has suddenly stopped working(!) - it works in the sam/delete branch I pushed earlier (the other PR I sent yall) but something in this branch has broken it and I can't figure it out
(the console gives a 'patch success' message, but the changes don't reflect in the log -- neither in the automatic refresh nor if you manually refresh it after)

Also I'm running into a bug where - if you use the show photo modal and then sign out - when you sign back in the sign-in modal shows the show-photo modal's content (I tried emptying the modal body contents as well in a ton of different places but to no avail)
I think I'm going to just forget about using modals for the sign up/sign in views and do it on page instead - but I thought I'd show you here in case y'all spotted anything

The update photo issue though I really need help on - thank you!

### CODE REVIEW

File: assets/scripts/auth/ui.js [hunk 1]
```diff
@@ -3,26 +3,30 @@
...
 const onSignUpFailure = function (response) {
-  $('#sign-up-message').text('sign up failure')
+  $('#signup-message').text('sign up failure')
   $('#sign-up').trigger('reset')
 }
 
 const onSignInSuccess = function (response) {
-  $('.sign-in-wrapper').addClass('hidden')
-  $('.main-content').removeClass('hidden')
+  $('#show-modal-body').empty()
```
Comment (JakeSeib) [inline assets/scripts/auth/ui.js]: Was this line just for debugging? Why empty the show-modal on sign in success?

File: index.html [hunk 2]
```diff
@@ -51,36 +84,47 @@ <h1>Subidos</h1>
...
+          <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLongTitle">Your Photo:</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div id="show-modal-body" class="modal-body"></div>
+              <div class="modal-footer">
```
Comment (JakeSeib) [inline index.html]: As far as I can tell, your show-photo modal has multiple headers and footers. The `show-display-owned.handlebars` already has its own header and footer. I'm not really sure what bootstrap does if you have a modal with multiple footers, but this could have something to do with your update not working.

File: assets/scripts/auth/ui.js [hunk 3]
```diff
@@ -37,10 +41,12 @@ const onChangePwFailure = function (response) {
 }
 
 const onSignOutSuccess = function (response) {
-  $('#sign-up-message').text('sign out success')
-  $('.sign-in-wrapper').removeClass('hidden')
-  $('.main-content').addClass('hidden')
-  $('#change-pw').trigger('reset')
+  store.user = null // wipes signed-in users data clean
+  $('.modal').modal('hide')
+  $('#show-modal-body').empty()
```
Comment (JakeSeib) [inline assets/scripts/auth/ui.js]: I'm pretty sure the issue you're trying to solve here is in `photo/ui.js`. You're clearing the show-modal-body alright, but `onShowPhotoSuccess` is setting the html of ALL elements with class `modal-content` (including the sign-in modal).

This was fine when we only had one modal, but at this point we need to change it to only set the html of the show-photo modal and leave the sign in/up ones alone.

Comment (JakeSeib) [reply assets/scripts/auth/ui.js]: If you do end up needing to mess around with other ways of hiding/clearing modals or their content though, you could look at either other modal methods:
https://getbootstrap.com/docs/4.0/components/modal/#methods

or some other jQuery methods, like .remove() or .detach():
https://www.w3schools.com/jquery/html_detach.asp
